docs/tooling/cli-guides/validate-primary-network.md

```diff
@@ -0,0 +1,64 @@
+# Be a Primary Network validator 
+
+This page will guide you on how to have your node validate Primary Network! Currently, we are only 
+supporting validating on Fuji network with Mainnet support coming in the near future!
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Prerequisites
+
+Before we begin, you will need to have:
+
+- Created a Cloud Server node as described [here](/docs/tooling/cli-guides/create-a-validator.md)
+- Node is bootstrapped to Primary Network (run `avalanche node status <clusterName>`to check 
+bootstrap status)
+- Stored key / Ledger with AVAX to pay for gas fess associated with adding node as Primary Network 
+and Subnet Validator transactions. Instructions on how to fund stored key on Fuji can be found
+[here](https://docs.avax.network/build/subnet/deploy/fuji-testnet-subnet#funding-the-key)
+
+## Be a Primary Network Validator
+
+Once the node is bootstrapped to Primary Network, we can now have the node be a Primary Network 
+Validator.
+
+To be a Primary Network Validator, run:
```

Comment:
Let's say the node belongs to the cluster myValidatorsCluster. To enable it to become a Primary Network Validator, run: